test/e2e/kubectl.go

```diff
@@ -238,6 +240,52 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		var nsFlag string
+		var rcName string
+
+		BeforeEach(func() {
+			nsFlag = fmt.Sprintf("--namespace=%v", ns)
+			rcName = "e2e-test-nginx-rc"
+		})
+
+		AfterEach(func() {
+			runKubectl("stop", "rc", rcName, nsFlag)
+		})
+
+		It("should create an rc from an image", func() {
+			image := "nginx"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			rc, err := c.ReplicationControllers(ns).Get(rcName)
+			if err != nil {
+				Failf("Failed getting rc %s: %v", rcName, err)
+			}
+			containers := rc.Spec.Template.Spec.Containers
+			if containers == nil || len(containers) != 1 || containers[0].Image != image {
+				Failf("Failed creating rc %s for 1 pod with expected image %s", rcName, image)
+			}
+
+			By("verifying the pod controlled by rc " + rcName + " was created")
+			label := labels.SelectorFromSet(labels.Set(map[string]string{"run": rcName}))
+			err = waitForPodsWithLabelRunning(c, ns, label)
```

Comment:
I tried to add `waitForPodsWithLabelRunning()` because `waitForRCPodsRunning()` uses the label `name=<replication name>` that this pod doesn't have. 
However, it didn't help, since `waitForPodsWithLabelRunning()` / `waitForRCPodsRunning()` uses similar call as what we do here (line 277) to get pods back before starting monitoring pods status:
```go
c.Pods(namespace).List(label, field)
// this will still get 0 pods so we can't monitor pod status
```

I think it shouldn't return 0 pods. Would it be a bug of `Client`?